### PR TITLE
Adding r-prais (and r-pcse required)

### PR DESCRIPTION
**Pull request by pn-alex**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/5eddbd7fc9d1502169089da06c3688d9759be978/recipes/example/meta.yaml#L64-L73) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Package does not ship static libraries. If static libraries are needed, [follow CFEP-18](https://github.com/conda-forge/cfep/blob/main/cfep-18.md).
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [ ] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [ ] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

File: recipes/r-prais/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+  home: https://github.com/franzmohr/prais
+  license: GPL-2.0-only
+  summary: The Prais-Winsten estimator (Prais & Winsten, 1954) takes into account AR(1) serial
+    correlation of the errors in a linear regression model. The procedure recursively
+    estimates the coefficients and the error autocorrelation of the specified model
+    until sufficient convergence of the AR(1) coefficient is attained.
+  license_family: GPL2
+  license_file:
+    - '{{ environ["PREFIX"] }}/lib/R/share/licenses/GPL-2'
+# Encoding: UTF-8
```
Comment (mfansler) [inline recipes/r-prais/meta.yaml:53]: Drop superfluous comment:

```suggestion
```

**Review by mfansler — COMMENTED** [review]
Thanks for submitting this! ❤️ 

Looks good, but appears it needs `r-pcse` to be added. Feel free to add that as part of this PR - the CI will automatically build them in the correct order. Alternatively, a separate PR is also fine.

**Comment by pn-alex** [comment]
@conda-forge/help-r, ready for review! :)

File: recipes/r-pcse/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+    - r-base
+
+test:
+  commands:
+    - $R -e "library('pcse')"           # [not win]
+    - "\"%R%\" -e \"library('pcse')\""  # [win]
+
+about:
+  home: https://CRAN.R-project.org/package=pcse
+  license: GPL-3
```
Comment (mfansler) [inline recipes/r-pcse/meta.yaml:39]: DESCRIPTION shows `License: GPL (>= 3)`, so

```suggestion
  license: GPL-3.0-or-later
```

**Review by mfansler — CHANGES_REQUESTED** [review]
License needs a minor adjustment; otherwise good.

**Pull request merged**